core-aam/index.html

```diff
@@ -548,18 +548,15 @@ <h2>Computed Role</h2>
 &lt;a href="#" role="button"&gt; &lt;!-- computedrole returns "button" --&gt;</pre
             >
           </aside>
-          <p>
-            When an element has a role but is not contained in the required context (for example, an orphaned `listitem` without the required accessible parent of role `list`), User Agents MUST ignore
-            the role token, and return the `computedrole` as if the ignored role token had not been included.
+          <p class="note">
+            When an element has a role but is not contained in the required context (for example, an orphaned `listitem` without the required accessible parent of role `list`), this is an authoring
+            error, but the user agent behavior is not specified as a single rule. For most roles, user agents can either recover the error by ignoring the role, or respect the author's intended role
+            in scenarios deemed by the implementation to be harmless. This unspecified permissiveness in how engines treat author role usage errors is occasionally overridden for a specific role in
+            other specs such as [[HTML-AAM]].
```

Comment:
```suggestion
            in scenarios deemed by the implementation to be harmless. Please note that this permissiveness in how engines treat author role errors might be overridden in a language-specific mapping document such as [[HTML-AAM]].
```

Reply:
@spectranaut The **Prettier** bot broke the thread (because of the line breaks it needlessly added) but this is in response to [your feedback suggestion](https://github.com/w3c/aria/pull/2284/files#r1683523037).

Reply:
Ok, this seems better to me!